### PR TITLE
Russian tokenizer

### PR DESCRIPTION
**Pull request by my-master**
* Russian tokenizer + lemmatizer
* pymorphy2 requirements

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -20,3 +20,5 @@ flask==0.12.2
 flasgger==0.6.6
 flask_cors==3.0.3
 scipy==1.0.0
+pymorphy2==0.8
```
[the remark this answers is not in the document]
Comment (my-master) [reply requirements.txt]: I have never used it. Should test first, but don't have time for it until May 7.

[the remark this answers is not in the document]
Comment (seliverstov) [reply requirements.txt]: Do you need to test it with some additional tests, or we can use our standard test suite?

[the remark this answers is not in the document]
Comment (my-master) [reply requirements.txt]: I mean additional tests. I should make sure that "fast" implementation predicts lemmas right and that there is no memory leak and cpu overloading.

**Comment by seliverstov** [comment]
Duplicate of #201

**Pull request closed without merging**